### PR TITLE
Added railroading support with broadcast!/2

### PR DESCRIPTION
**Pull request by saulecabrera**
Adds the functionality discussed in #48

### CODE REVIEW

**Comment by saulecabrera** [comment]
@Zorbash,

I'm not quite sure why the build is failing. I tried to reproduce the failures locally with the corresponding versions of Elixir & OTP, tried to re-trigger the build and the failures are not consistent. 

1st failure: failed with OTP 18.3 & Elixir 1.4.0 (tried locally and I was able to reproduce the failure only once).
2nd failure: failed with OTP 18.3 & Elixir 1.3.4 (in the first run of the build this failure was not reported).

The failures are also reported in parts of the code not modified by this PR (at least directly).

Maybe I'm missing something here? 
Let me know if you have any inputs on this. I'll try to investigate further.

**Comment by zorbash** [comment]
@saulecabrera The build failures are caused by known flappy tests (which I'll try to fix during the weekend) and are not related to your MR (restarted the failed builds).

File: lib/kitto/notifier.ex [hunk 1]
```diff
@@ -44,6 +44,7 @@ defmodule Kitto.Notifier do
   Emits a server-sent event to each of the active connections with the given
   topic and payload
   """
+  def broadcast!(data, topic) when is_atom(topic), do: broadcast!(topic, data)
```
Comment (zorbash) [inline lib/kitto/notifier.ex:47]: I was going to ask you to add a test for this, but we don't have a test file for this module.

**Comment by zorbash** [comment]
Looks good to me. Thanks a lot for taking the time to submit this PR.

File: lib/kitto/notifier.ex (hunk 1 above)
Comment (zorbash) [reply lib/kitto/notifier.ex:47]: ..yet 😉

**Pull request merged**